inst/@sym/subsasgn.m

```diff
@@ -513,8 +513,78 @@
 %! a(1) = [];
 %! assert( isequal( a, b));
 
+%!test
+%! % indexing with a dimension more than 2
```

Comment:
I actually got confused while choosing the label. This test actually is for indexing with a mXn matrix where either one of m or n is more than 2. So probably something along the lines of "indexing with a matrix with size of a dimension more than 2"!?